### DISCUSSION TITLE
How to add certificate to openshift/microshift instance running on Windows

### DISCUSSION THREAD

**Discussion by darkedges** [crc-org/crc#3656] [Q&A]
It seems if I do `crc ip` it returns `127.0.0.1` and if I try to ssh to it it will time out.

How can I add a certificate to the underlying hyper-v virtual machine? Or is there another way to make all registries insecure?

**Comment by praveenkumar** [comment]
@darkedges Try using https://github.com/crc-org/crc/wiki/Debugging-guide 
```
If you use vsock network mode, the IP is 127.0.0.1 and the port is 2222. On Windows, the relevant SSH keys is in C:\Users\%USERNAME%\.crc\machines\crc\id_ecdsa
```
